cheatsheets_draft/OAuth_Cheat_Sheet.md

```diff
@@ -29,7 +30,23 @@ This Cheat Sheet provides guidance for application builders on how to deploy OAu
 
 ## Terminology
 
-Here go explanation of this, that, and the other term...
+In this section, the most important terms for OAuth 2.0 will be shortly explained:
+
+### Roles
+
+OAuth 2.0 defines these four most important roles:
+
+1. **Resource Owner**
+The Resource Owner is the person or entitiy that can grant access to a certain resource. Typically, this is the end-user of an application.
+
+2. **Resource Server**
+The Resource Server is the server hosting the protected resource and deciding whether access (with an OAuth 2.0 Access Token) is accepted or not. This is the application or API you want to access.
+
+3. **Client**
+The Client is the application that accesses a protected resource (on the Resoure Server) on behalf and with the authorization of a Resource Owner (end-user).
+
+4. **Authorization Server**
+The Authorization Server is the service that authenticates the Resource Owner and issues Access Tokens to the Client after authorization by the Resource Owner. This could be your central Single-Sign-On or Authorization Service.
```

Comment:
This currently reads: `The Authorization Server is... This could be your central SSO or Authorization Server.` and sounds a bit repetitive.

Maybe it would read better as:

`The Authorization Server is... This could be your central SSO or Identity Provider.`

?

Reply:
Fully agree, I missed the repetition :-) Well, I wanted to say 'an authorization service' (from outside of OAuth terminology :-)) - as opposed to SSO as a typical authentication service. Any better term for this? Otherwise, I'll got with your suggestion, as it's typically done by SSO/IdP anyway...

Reply:
I pushed an update but feel free to edit it further!